### ISSUE TITLE
Split Button: Aria Label not correctly working

### ISSUE THREAD

**Issue by tomhuber96** [primefaces/primeng#15342]
### Describe the bug

The ButtonProps and MenuButtonProps are not correctly defined and cannot be used currently. 

They define a property 'ariaLabel', but in the code in is accessed as 'aria-label'. 
The same problem exists for the other properties of MenuButtonProps. 

For reference see: 
- https://github.com/primefaces/primeng/blob/17.14.1/src/app/components/splitbutton/splitbutton.ts#L74
- https://github.com/primefaces/primeng/blob/17.14.1/src/app/components/splitbutton/splitbutton.interface.ts

I tested it in v17.13, but as you see in the links above, it should still exist in 17.14.1.

 

### Environment

I am using angular 17

### Angular version

17

### PrimeNG version

17.13.0

### Build / Runtime

Angular CLI App

### Language

TypeScript

### Node version (for AoT issues node --version)

10
